examples/cli/src/main/java/io/unitycatalog/cli/AuthCli.java

```diff
@@ -75,24 +74,21 @@ private static String exchange(ApiClient apiClient, JSONObject json)
   private static String doExchange(ApiClient apiClient, Map<String, String> login)
       throws JsonProcessingException, ApiException {
 
-    URI tokensEndpoint = URI.create(apiClient.getBaseUri() + "/auth/tokens");
+    URI endpoint = URI.create(apiClient.getBaseUri() + "/auth/tokens");
 
-    StringBuilder builder = new StringBuilder();
-    builder
-        .append("grant_type=")
-        .append(URLEncoder.encode(GrantTypes.TOKEN_EXCHANGE, StandardCharsets.UTF_8))
-        .append("&requested_token_type=")
-        .append(URLEncoder.encode(TokenTypes.ACCESS, StandardCharsets.UTF_8))
-        .append("&subject_token_type=")
-        .append(URLEncoder.encode(TokenTypes.ID, StandardCharsets.UTF_8))
-        .append("&subject_token=")
-        .append(URLEncoder.encode(login.get("identityToken"), StandardCharsets.UTF_8));
+    String body =
+        Oauth2CliExchange.URLEncodedForm.ofMap(
+            Map.ofEntries(
+                entry("grant_type", GrantType.TOKEN_EXCHANGE.getValue()),
```

Comment:
Is there not a way to use static strings or enums instead of the strings here like `"grant_type"` ? @jaceklaskowski do you have any thoughts/opinions here?

Reply:
I don't have a strong opinion on this matter, but as @creechy suggested in the source code comments, defining it as an enum in a shared utility could be a good approach. However, since the constant field names are not used outside of the CLI implementation, I thought implementing it as an enum might not be strictly necessary. That said, I’m happy to leave the final decision to the team.

Reply:
While true, I can see the grant type being used outside of just the CLI implementation  and the `AuthService` especially as extend our authentication features.  While not wanting to be a blocker, IMHO, I tend to agree with @creechy that we should probably used static strings or enums if possible, eh?!